src/test/scala/za/co/absa/hyperdrive/trigger/models/tables/SearchableTableQueryTest.scala

```diff
@@ -243,6 +247,38 @@ class SearchableTableQueryTest extends FlatSpec with Matchers with BeforeAndAfte
     result.total shouldBe 0
   }
 
+  "the multiple-attributes filter" should "find values exactly equal to the List of strings" in {
+    val filter = ContainsMultipleFilterAttributes(field = TestSearchableTableFieldNames.stringField3, values = List("foo", "bar"))
+    val searchRequest = TableSearchRequest(
+      containsMultipleFilterAttributes = Some(Seq(filter)),
+      sort = None,
+      from = 0,
+      size = 50
+    )
+
+    val result = await(db.run(underTest.search(searchRequest)))
+
+    val expected = TestSearchableData.testSearchableEntities.filter(data => data.stringValue3.contains("foo") || data.stringValue3.contains("bar"))
```

Comment:
I have changed it to == instead of .equals()
Let me know what you think

Reply:
Yes `==` is preferred over `.equals` in Scala, because it handles `null` properly. https://stackoverflow.com/questions/7681161/whats-the-difference-between-and-equals-in-scala